package.json

```diff
@@ -2,40 +2,80 @@
   "name": "babili-webpack-plugin",
   "version": "0.1.2",
   "description": "Babili Plugin for Webpack",
-  "main": "lib/index.js",
+  "author": "boopathi",
+  "license": "MIT",
+  "main": "dist/cjs.js",
   "files": [
-    "lib"
+    "dist"
   ],
   "scripts": {
-    "build": "babel src --out-dir lib",
+    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "watch": "babel src --out-dir lib --watch",
-    "lint": "eslint src/ test/*.js",
+    "lint": "eslint --cache src test",
     "fix": "eslint src/ test/*.js --fix",
-    "test": "mocha --compilers js:babel-core/register"
+    "test": "jest",
+    "webpack-defaults": "webpack-defaults",
+    "start": "npm run build -- -w",
+    "appveyor:test": "npm run test",
+    "clean": "del-cli dist",
+    "lint-staged": "lint-staged",
+    "prebuild": "npm run clean",
+    "prepublish": "npm run build",
+    "release": "standard-version",
+    "security": "nsp check",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
+    "travis:lint": "npm run lint && npm run security",
+    "travis:test": "npm run test -- --runInBand",
+    "travis:coverage": "npm run test:coverage -- --runInBand"
   },
-  "keywords": [
-    "webpack",
-    "webpack-plugin",
-    "babel",
-    "babili",
-    "minify"
-  ],
-  "repository": "https://github.com/webpack-contrib/babili-webpack-plugin",
-  "author": "boopathi",
-  "license": "MIT",
   "dependencies": {
     "babel-core": "^6.24.1",
     "babel-preset-babili": "^0.1.4",
     "webpack-sources": "^1.0.1"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
+    "babel-jest": "^20.0.3",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-polyfill": "^6.23.0",
     "babel-preset-env": "^1.5.1",
+    "cross-env": "^5.0.1",
+    "del-cli": "^1.1.0",
     "eslint": "^4.1.1",
+    "eslint-config-webpack": "^1.2.5",
+    "eslint-plugin-import": "^2.7.0",
     "expect": "^1.20.2",
+    "jest": "^20.0.4",
+    "lint-staged": "^4.0.1",
     "mocha": "^3.4.1",
+    "nsp": "^2.6.3",
+    "pre-commit": "^1.2.2",
```

Comment:
Open an issue in `webpack-defaults` about this please, otherwise any update to `webpack-defaults` will add this back again and the boilerplate sync will be useless/cumbersome